docs/install/index.md

```diff
@@ -38,16 +38,25 @@ Scaffold your InvenioRDM instance. Replace ``<version>`` with the version you wa
 - LTS release (for production systems): ``v9.1``
 - STS release (for feature previews): ``v11.0``
 
-```
+```console
 invenio-cli init rdm -c <version>
 # e.g:
 invenio-cli init rdm -c v9.1
 ```
 
 You will be asked several questions. If in doubt, choose the default.
 
+
 #### [4. Build, setup and run](build-setup-run.md)
 
+Now the scaffoling is complete, it is time to check the development requirements
+```console
+cd my-site/
+invenio-cli check-requirements --development
+```
+This will check the python, nodejs and npm versions that depend on the version of InvenioRDM that has been selected in the previous step
```

Comment:
```suggestion
This will check the versions of the packages that depend on the selected version of InvenioRDM
```

Reply:
Works for me (with an added period at the end of the sentence) :+1:

Reply:
I went for the suggestion of removing the sentence completely. If you prefer this shorter version, that's also fine for me